tools/cldr-code/src/main/java/org/unicode/cldr/util/DtdType.java

```diff
@@ -29,16 +29,22 @@ public enum DtdType {
             "transforms",
             "validity"),
     ldmlBCP47("common/dtd/ldmlBCP47.dtd", "1.7.2", null, "bcp47"),
-    // keyboard 3.0
-    keyboard("keyboards/dtd/ldmlKeyboard.dtd", "44.0", null, "../keyboards/3.0"),
-    keyboardTest("keyboards/dtd/ldmlKeyboardTest.dtd", "44.0", null, "../keyboards/test");
+    keyboard2("keyboards/dtd/ldmlKeyboard2.dtd", "44.0", null, "../keyboards/3.0"),
```

Comment:
Yeah, shouldn't it be called ldmlKeyboard3.dtd? :-)

Reply:
3? right…

Reply:
Can be either way, but probably would make sense if it was unified, either ldmlKeboard2 and keyboards/2.0 or ldmlKeyboard3 and keyboards/3.0